tests/versioned/apollo-federation/service-definition-and-healthcheck-filtering.test.js

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const utils = require('@newrelic/test-utilities')
+const createPlugin = require('../../../lib/create-plugin')
+
+const { loadLibraries, loadGateway } = require('./federated-gateway-server-setup')
+
+tap.test('Capture/Ignore Service Definition and Health Check ' +
+  'query transaction from sub-graph servers', (t) => {
+  t.autoend()
+
+  let gatewayServer = null
+  let libraryServer = null
+  let helper = null
+  let agentConfig = null
+
+  t.afterEach(() => {
+    helper.unload()
+
+    gatewayServer = null
+    libraryServer = null
+    helper = null
+    agentConfig = null
+  })
+
+  t.test('Should not ignore Service Definiion query by default', async (t) => {
```

Comment:
shouldn't this title be `Should ignore Service Definition query by default`?

Reply:
🤦  yes